src/index.ts

```diff
@@ -14,4 +14,5 @@ export * from './main/menu';
 
 // Assign `global` so gas-webpack-plugin will generate a stub for `onOpen` in the top-level scope. A top-level
 // declaration is needed for Google Apps Script to detect the function, so it is available as a trigger.
-global.onOpen = onOpen;
```

Comment:
i suspect onOpen might be something google sheets or google apps scripts actually uses? like when the sheet opens, it'll call this function?

Reply:
Yeah that's precisely what it does. When the sheet opens it runs the logic that creates the menu:

```
export const onOpen = (e: OpenEvent) => {
  const tabSyncMenu: GoogleAppsScript.Base.Menu = SpreadsheetApp.getUi()
    .createMenu('OneStop Utils');

  tabSyncMenu.addToUi();
}
```

I guess we could keep it so that the menu gets recreated every time someone opens the OneStop. However, I feel like that would produce even more unecessary executions than when running  `updateManualTriggerMenuUiOnOneStop()` every time `importOnestopToBasecamp()` runs which I now think is also unnecessary

Reply:
so if onOpen isn't called, then no menu is created? or is it just the last menu from the last execution?

i do see your point though; are you able to tell if the sheet loads more slowly as a result? if there's no noticeable difference, it might be ok to have this run on every open, even if unnecessary. but if there's any noticeable slow down, i'm in favor of saving on these extra executions like you're bringing up

Reply:
I'll have to dig into whether onOpen persists. I have a feeling that it doesn't because when I reload my sheet the custom menu is not there but then pops up after 3 seconds

Reply:
Okay, it actually doesn't persist. It needs to get added each time onOpen. I'll change that.

Reply:
@andrewc5716  Okay I think this PR should be good now?